packages/aragon-cli/src/commands/apm_cmds/publish.js

```diff
@@ -107,8 +107,50 @@ exports.builder = function(yargs) {
     })
 }
 
+const getMajor = version => version.split('.')[0]
+
+function getFunctionsAbi(functions, abi) {
+  functions.forEach(f => {
+    f.abi = abi.find(method => method.name === f.sig.match(/(.*)\(/)[1])
+  })
+}
+
+async function getDeprecatedFunctions(apm, artifact) {
+  let deprecated = {}
+  try {
+    let deprecatedOnVersion = []
```

Comment:
Let's pull this into the `forEach()` as an inner variable (so we don't have to reset it manually).